### PR TITLE
🐛 [WIP]  Fix docker load balancer name for length restriction 

### PR DESCRIPTION
**Pull request by killianmuldoon**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: test/infrastructure/docker/internal/docker/loadbalancer.go [hunk 1]
```diff
@@ -101,7 +101,11 @@ func getLoadBalancerImage(dockerCluster *infrav1.DockerCluster) string {
 
 // ContainerName is the name of the docker container with the load balancer.
 func (s *LoadBalancer) containerName() string {
-	return fmt.Sprintf("%s-lb", s.name)
+	lbName := s.name
+	if len(s.name) > 57 {
+		lbName = s.name[:57]
+	}
+	return fmt.Sprintf("%s-lb", lbName)
```
Comment (fabriziopandini) [inline test/infrastructure/docker/internal/docker/loadbalancer.go:108]: Let's comment why are we doing this for future reference.
Also, should we do the same for the containers hosting CP and workers?

Comment (killianmuldoon) [reply test/infrastructure/docker/internal/docker/loadbalancer.go:108]: Yeah we need to - but it's a little bit more complicated. I'm working on a fix there.

**Comment by killianmuldoon** [comment]
/hold 

Still WIP

**Comment by vincepri** [comment]
Do we still want to do this?

**Comment by killianmuldoon** [comment]
>Do we still want to do this?

Yeah - I'll get back to it at some point, just fallen off my to-do list :slightly_smiling_face:

**Review by killianmuldoon — COMMENTED** [review]
/close

**Pull request closed without merging**